app/views/integrations/index.html+turbo_frame.erb

```diff
@@ -1,5 +1,5 @@
 <% component = V2::EnhancedTurboFrameComponent.new(dom_id(@app, :edit_app)) %>
 <% component.with_loading_indicator %>
 <%= render component do %>
-  <%= render V2::IntegrationListComponent.new(@app, @integrations_by_categories, pre_open_category: @pre_open_category) %>
+  <%= render V2::IntegrationListComponent.new(@app, @integrations_by_categories, @existing_provider_integration, pre_open_category: @pre_open_category) %>
```

Comment:
**question**: what is this third parameter? the component initializer does not seem have changed?